Assets/Scripts/Globals.cs

```diff
@@ -5,9 +5,11 @@
 public class Globals : MonoBehaviour
 {
     // Start is called before the first frame update
-    void Start()
+    public static class Guns
     {
-        
+        public static Dictionary<int, Bullet> bulletTypes = new Dictionary<int, Bullet>(){

```

Comment:
I'm not convinced that having separate bullet types is the correct path. Another method is to have one Bullet class with a list of modifications. Let's chat about this.